ucm2/sof/product_configs/AAEON/UPX-TGL01.conf

```diff
@@ -0,0 +1,20 @@
+# Add customization of IIR, FIR, and DRC
```

Comment:
Nitpick: commit message: s/TO/To and I would reword it to be a bit more cleaner.

Reply:
I rewrote it and placed all defines as comments to not change anything but show how it would be done.